server/covmanager/cron.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import requests
+
+from celeryconf import app
+from dateutil.relativedelta import relativedelta, MO
+from django.conf import settings
+from django.db.models import Q
+
+from crashmanager.models import Client
+from .models import Collection, Repository
+from .tasks import aggregate_coverage_data
```

Comment:
I believe this will have to move inside the `create_weekly_report_mc` function to prevent a circular import loop.
`tasks.py` should import `cron.py` since celery only looks in `tasks.py` for tasks. In EC2SpotManager we have:

    from . import cron  # noqa ensure cron tasks get registered